docs/operator-manual/upgrading/3.0-3.1.md

```diff
@@ -5,3 +5,17 @@
 The `--staticassets` directory in the API server (`/app/shared` by default) is now protected against out-of-bounds 
 symlinks. This is to help protect against symlink attacks. If you have any symlinks in your `--staticassets` directory
 to a location outside the directory, they will return a 500 error starting with 3.1.
+
+## Breaking Changes
+
+### Kube versions
+
+Until v3.0, Argo CD removed `+` identifier from `kubeVersions` in Helm, Kustomize and Plugins. For example, if Argo CD receive `kubeVersions` as vX.Y.Z+, we convert to vX.Y.Z internally. After v3.0, the internal conversation is entirely removed.
+
+#### Detection
+
+To detect if you are affected, check the `kubeVersions` you'are using. If you use `kubeVersions` including `+` identifier, the application must be failed when you upgrade from v3.0 to v3.1.
+
+#### Remediation
+
+This problem is that Argo CD removes no longer `+` identifier from `kubeVersions. Your application has already used `kubeVersions` triming `+` identifier, so it just works correctlly by removing `+` identifier. 
```

Comment:
```suggestion
The plus symbol was originally removed [because Helm did not support it](https://github.com/argoproj/argo-cd/issues/2303). Helm no longer enforces this restriction, so apps should work fine even if your kubeVersions include `+`.

However, since Argo CD now sends unmodified kubeVersions to Helm, the generated manifests may differ due to the changed kubeVersions. For apps which require extra caution, it may be advisable to disable auto-sync, upgrade, check any diffs, and then reenable auto-sync.
```